apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -90,6 +95,59 @@ export class VersionManager {
       });
       changeManager.updateChangelog(!!shouldCommit);
     }
+
+    this._updateTarballIntegrities();
+  }
+
+  private _updateTarballIntegrities(): void {
```

Comment:
You will also need to run `RepoStateFile.Refresh()` (or whatever I called the method heh) to ensure the hash is correctly updated